### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Strangeal**
### Form validation

- Added email validation in js

### CODE REVIEW

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Strangeal, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you have done individually but this project is pair-programming. please do it with your partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -150,3 +150,48 @@ cardButton.forEach((each) => each.addEventListener('click', function OpenPopUp()
...
+    setError(email, 'Provide a valid email address');
+  } else {
+    setSuccess(email);
+  }
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  validateInputs();
+});
```
Comment (medaminedev66) [inline index.js:186]: - [x] Even though I only used lowercase characters when entering an email, your form is still showing that it is incorrect. Please ensure If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

File: index.html [hunk 2]
```diff
@@ -142,12 +142,12 @@ <h2>About Myself</h2>
...
                 <form action="https://formspree.io/f/myyvzkjz" method="POST" class="form-input">
-                <div class="input-field error">
+                <div class="input-field">
                   <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" aria-labelledby="name" required>
                 </div>
-                  <div class="input-field success">
+                  <div class="input-field">
                     <input type="email" name="Email" id="email" placeholder="Enter your email" aria-labelledby="email" required>
-                    <small style="color: red; font-size: 30px;"></small>
+                    <span class="error"></span>
```
Comment (medaminedev66) [inline index.html]: - [x] Following the project requirements, the error message should show up close to the "submit" button.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi team  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional.
- Git/GitHub flow best practices have been followed.
- There are no linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Strangeal,😃 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🥇 
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**